### ISSUE TITLE
Update homology document

### ISSUE THREAD

**Issue by mycrobe** [warelab/gramene-mongodb#36]
In #21, the homology subdocument was tidied up and now looks something like this:
```
{
    "_id": "GLYMA01G22160",
    …
    "homology": {
        "gene_tree": {
            "id": "EPlGT00140000003276",
            "root_taxon_id": 3193
        },
        "representative": {
            "closest": {
                "id": "MTR_5g053950",
                "taxon_id": 3880,
                "description": "allene oxide cyclase"
            },
            "model": {
                "id": "AT3G25780",
                "taxon_id": 3702,
                "name": "AOC3",
                "description": "Allene oxide cyclase 3, chloroplastic [Source:UniProtKB/Swiss-Prot;Acc:Q9LS01]"
            }
        },
        "within_species_paralog": [
            "GLYMA02G11020",
            …
        ],
        "ortholog_one2many": [
            "MTR_5g053950",
            …
        ],
        "ortholog_many2many": [
            "AT1G13280",
            …
        ]
    }
}
```

Unfortunately, we are mixing two different kinds of entity within the same doc: 
* Structured docs under the `gene_tree` and `representative` keys that describe the homology of this gene
* Arrays of gene id lists that enumerate homologs of various classes.

We should encapsulate those arrays into their own subdoc. 
```
{
    "_id": "GLYMA01G22160",
    …
    "homology": {
        "gene_tree": {
            "id": "EPlGT00140000003276",
            "root_taxon_id": 3193
        },
        "representative": {
            "closest": {
                "id": "MTR_5g053950",
                "taxon_id": 3880,
                "description": "allene oxide cyclase"
            },
            "model": {
                "id": "AT3G25780",
                "taxon_id": 3702,
                "name": "AOC3",
                "description": "Allene oxide cyclase 3, chloroplastic [Source:UniProtKB/Swiss-Prot;Acc:Q9LS01]"
            }
        },
        "homologous_genes": {
            "within_species_paralog": [
                  "GLYMA02G11020",
                  …
            ],
            "ortholog_one2many": [
                  "MTR_5g053950",
                  …
            ],
            "ortholog_many2many": [
                  "AT1G13280",
                  …
            ]
        ]
    }
}
```

**Comment by ajo2995** [comment]
I can make this change for release 50 if you want...

**Comment by mycrobe** [comment]
…OK.

**Comment by mycrobe** [comment]
This has been fixed because warelab/gramene-genetree-vis#24
